### PR TITLE
reeact tabs

### PR DESCRIPTION
**Pull request by Mohameddahi**
[DEMO LINK](https://mohameddahi.github.io/react_tabs/)

### CODE REVIEW

**Review by lukacsja — APPROVED** [review]
🥇 i have nothing to add, GJ

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -1 +1,61 @@
...
+  onTabSelected,
+  selectedTabId,
+
+}) => {
+  const selectedTab: TabsType = tabs
+    .find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div className="section">
+
```
Comment (naziamoff) [inline src/components/Tabs/Tabs.tsx:26]: no empty spaces between parent and child elements

**Review by naziamoff — APPROVED** [review]
looks good 👍

**Comment by Mohameddahi** [comment]
>

**Pull request closed without merging**